source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -83,13 +86,55 @@ the SRV record. For example, when using the non-FQDN ``servername`` and the
 default domain set to ``example.com``, a Client should end up requesting the
 SRV record for ``servername.example.com``.
 
+Default Connection String Options
+---------------------------------
+
+A Client MUST also query the DNS server for TXT records. If available, these
+TXT records provide default connection string options. In most cases only one
+TXT record is necessary, unless the default options exceed the maximum length
+of 255 characters that TXT records allow for. To allow for this case, a Client
+MUST allow for multiple TXT records for the same host name and consider the
+options in all of them.
+
+Information returned with each TXT record is a simple URI string, just like
+the options in a connection string.
+
+Although options in a connection string are separated by a ``&``, this
+character MUST NOT be present to separate options between TXT record strings
+as a separation is implied by using multiple TXT records.
```

Comment:
I approve this entire change, but I request another PR in which you justify this choice. I propose you add in the justifications section:

> Why Are Multiple Key-Value Pairs Allowed In One TXT Record?
> 
> One could imagine an alternative design in which each TXT record would allow only one URI option. No "&" character would be allowed as a delimiter within TXT records.
>
> In this spec we allow multiple key-value pairs within one TXT record, delimited by "&", because it will be common for all options to fit in a single 255-character TXT record, and it is much more convenient to configure one record in this case than to configure several.

Reply:
OK, will make one, and *also* mention that in some cases (readPreferenceTags) it is actually required to specify them in the same TXT record, as otherwise you can't guarantee order.

Reply:
Actually, I added it to this PR, as on its own this text makes no sense, and this PR hasn't been merged yet because @mbroadst has not approved it yet.

Reply:
yeah I'm into this behavior, but it will definitely be something that is asked after

Reply:
It's part of the last commit that I pushed here.